packages/material-ui-styles/src/getStylesCreator/getStylesCreator.js

```diff
@@ -1,6 +1,38 @@
 import { deepmerge } from '@material-ui/utils';
+import MuiError from '@material-ui/utils/macros/MuiError.macro';
 import noopTheme from './noopTheme';
 
+// TODO: remove this once the capiitalize is moved to the @material-ui/utils package
+export function capitalize(string) {
+  if (typeof string !== 'string') {
+    throw new MuiError('Material-UI: capitalize(string) expects a string argument.');
+  }
+
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+const isEmpty = (string) => {
```

Comment:
An arbitrary coding style, we use functions for top-level scope an arrow for nested ones

```suggestion
function isEmpty(string) {
```